external/anomaly/tests/anomaly_common.py

```diff
@@ -107,3 +112,18 @@ def get_anomaly_domain_test_action_classes(
         OTETestNNCFExportEvaluationAction,
         OTETestNNCFGraphAction,
     ]
+
+
+def get_dummy_compressed_model(task: NNCFTask) -> "NNCFNetwork":
+    """
+    Return compressed model without initialization
+    """
+    from anomalib.utils.callbacks.nncf.utils import wrap_nncf_model
```

Comment:
What's the reason for importing this here?

Reply:
@ashwinvaidya17 copypaste from other algorithms, maybe to avoid building nncf extensions when it do not used.

Reply:
Okay, thanks for the clarification.